src/aihwkit/utils/analog_info.py

```diff
@@ -0,0 +1,287 @@
+"""Analog Information utility.
+
+This module prints relevant information about the model and its analog execution.
+"""
+
+from functools import reduce
+import operator
+from typing import Optional, Any, List
+
+# Imports from PyTorch.
+import torch
+from torch import Tensor
+from torch import nn
+
+from aihwkit.nn.modules.base import AnalogModuleBase, RPUConfigAlias
+from aihwkit.nn.modules.conv_mapped import _AnalogConvNdMapped
+from aihwkit.nn.modules.linear_mapped import AnalogLinearMapped
+from aihwkit.nn.modules.conv import _AnalogConvNd
+from aihwkit.nn import AnalogLinear
+from aihwkit.simulator.tiles import BaseTile
+
+COLUMN_DEFINITIONS = ["Layer Information", "Tile Information"]
+
+COLUMN_NAMES = {
+    "name": (0, "Layer Name"),
+    "isanalog": (0, "Is Analog"),
+    "input_size": (0, "In Shape"),
+    "output_size": (0, "Out Shape"),
+    "kernel_size": (0, "Kernel Shape"),
+    "num_tiles": (0, "# of Tiles"),
+    # "macs": "Multi_Adds",
+    "log_in_size": (1, "in_size (log)"),
+    "log_out_size": (1, "out_size (log)"),
```

Comment:
I would prefer if one would use a single `log_shape` instead of two columns for in / out size:   `Log. tile shape (out, in)` could be good title and then display the shape `(out, in)`. 
Similarly one could add a column with 'Phys. tile shape (out, in)`

Reply:
What should I print for the physical shape when the layer is not mapped ?

Reply:
Just the logical size

Reply:
Just fixed it and added Phys. tile shape and Log. tile shape.

Reply:
great!